src/index.js

```diff
@@ -0,0 +1,21 @@
+import './style.css';
+import './to-go-bg.jpg';
+
+const taskInput = document.getElementById('input-task');
+const addBtn = document.getElementById('add-task');
+
+class Todo {
+  constructor(description, index, completed = false) {
+    this.description = description;
+    this.index = index;
+    this.completed = completed;
+  }
+}
+
+addBtn.addEventListener('click', () => {
+  if (taskInput.value !== '') {
+    const todolist = new Todo(taskInput.value, 0);
+    localStorage.setItem('todolists', JSON.stringify(todolist));
+    taskInput.value = '';
+  }
+});
```

Comment:
- [ ] the project should have an array of some simple to do tasks (array of objects) with each object containing three keys - description [string], completed [bool], index: [number].
- [ ]your project should also have a function that iterates over the tasks array and populates the HTML list item element for each task